File: Java Code/Myfirstjava/src/RunningSum.java
class Solution {
    public int[] runningSum(int[] nums) {
        int temp=0;
        // int n = nums.length;
        for(int i=0;i<nums.length;i++){
            temp = temp+nums[i];
            nums[i]=temp;
        }
        return(nums);
    }
}

File: Leetcode/running sum.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        # nums = list(map(int,input().split()))
        nsum = 0
        outnum = []
        for i in nums:
            nsum+=i
            outnum.append(nsum)
        return(outnum)

    